api/commands/prune.ts

```diff
@@ -77,5 +77,12 @@ function computeIncludedActionNames(
     }
   }
 
+  // Add auto assertions
+  [...compiledGraph.assertions].forEach(assertion => {
+    if (includedActionNames.has(utils.targetToName(assertion.parent))) {
```

Comment:
for extra credit: change `includedActionNames` into a `StringifiedSet<dataform.ITarget>` so this stuff isn't necessary.

Reply:
I got started on this, but a lot of things need to be transitioned from `name` to `Target`; will do in another PR